cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java

```diff
@@ -0,0 +1,277 @@
+                .with("pk.fields", "id")
+                .with("pk.mode", "record_key")
+                .with("delete.enabled", "true")
+                .with("auto.evolve", "true")
+                .with("value.converter", "org.apache.kafka.connect.json.JsonConverter")
+                .with("value.converter.schemas.enable", "true")
+                .with("key.converter", "org.apache.kafka.connect.json.JsonConverter")
+                .with("key.converter.schemas.enable", "true");
+
+        debeziumContainer.registerConnector("my-connector", connector);
+
+        HOST_ADDRESS = InetAddress.getLocalHost().getHostAddress();
+        TestHelper.setHost(HOST_ADDRESS);
+        BOOTSTRAP_SERVER = kafka.getNetworkAliases().get(0) + ":9092";
+        TestHelper.setBootstrapServer(BOOTSTRAP_SERVER);
+        String createTableSql = "CREATE TABLE IF NOT EXISTS test_table (id int primary key, first_name varchar(30), last_name varchar(50), days_worked double precision)";
+        TestHelper.execute(createTableSql);
+        String deleteFromTableSql = "DELETE FROM test_table";
+        TestHelper.execute(deleteFromTableSql);
+
+        // Initialise expected_data.
+
+        for (int i = 0; i < recordsInserted; i++) {
+            Map<String, Object> expected_record = new LinkedHashMap<String, Object>();
+            expected_record.put("id", new Integer(i));
+            expected_record.put("first_name", new String("first_" + i));
+            expected_record.put("last_name", new String("last_" + i));
+            expected_record.put("days_worked", new Double(23.45));
+            expected_data.add(expected_record);
+        }
+
+        cdcContainer = TestHelper.getCdcsdkContainer();
+        cdcContainer.withNetwork(network);
+        cdcContainer.start();
+        System.out.println("CDCSDK container created at: {}"
+                + cdcContainer.getContainerInfo().getNetworkSettings().getNetworks().entrySet().stream().findFirst().get().getValue().getIpAddress());
+
+        Thread.sleep(10000);
+
+        // Insert records in YB.
+
+        for (int i = 0; i < recordsInserted; ++i) {
+            String insertSql = String.format("INSERT INTO test_table VALUES (%d, '%s', '%s', %f);", i, "first_" + i,
+                    "last_" + i, 23.45);
+            TestHelper.execute(insertSql);
+        }
+        Thread.sleep(60000);
+
+    }
+
+    @AfterAll
+    public static void afterClass() throws Exception {
+        Path path_cdcserver = Paths.get("/home/ishaamoncar/test-log.txt");
+        Files.writeString(path_cdcserver, cdcContainer.getLogs(), StandardCharsets.UTF_8);
+
+        Path path_kafka = Paths.get("/home/ishaamoncar/test-log-kafka.txt");
+        Files.writeString(path_kafka, kafka.getLogs(), StandardCharsets.UTF_8);
+
+        Path path_connect = Paths.get("/home/ishaamoncar/test-log-connect.txt");
+        Files.writeString(path_connect, debeziumContainer.getLogs(), StandardCharsets.UTF_8);
```

Comment:
We may not need all these write to files now. I am anyway planning to add them as a part of test logs in the next iteration.